### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by mkbmain**

### CODE REVIEW

File: src/FileSigChecker.csproj [hunk 1]
```diff
@@ -1,14 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0-windows</TargetFramework>
```
Comment (mkbmain) [inline src/FileSigChecker.csproj:4]: Should now work on linux and windows

File: src/FileSigChecker.cs [hunk 2]
```diff
@@ -3,77 +3,73 @@
...
-		private static string _helpMessage {get; set;}
-
-		// Main method.
-		static void Main(string[] args)
-		{
-		    _extFileName = $@"{Application.StartupPath}ext_list.txt";
-		    _helpMessage =@" sage of file extension tool:
+    class FileTypeChecker
+    {
+        private const string _extFileName = @"ext_list.txt";
```
Comment (mkbmain) [inline src/FileSigChecker.cs:12]: const by default will be in startup path

File: src/FileSigChecker.cs [hunk 3]
```diff
@@ -86,122 +82,131 @@ public static void ColorConsoleWriteLine(ConsoleColor color, object data)
...
+        public static string GetHex(string fileName)
+        {
+            string outHex = "";
+            try
+            {
+                if (File.Exists(fileName))
+                {
+                    using (var stream = File.OpenRead(fileName))
+                    {
+                        int size = stream.Length < 50 ? (int)stream.Length : 50;
```
Comment (mkbmain) [inline src/FileSigChecker.cs:142]: will take 50 or if file is smaller than 50 just take the length of the file

Comment (0x78654C) [reply src/FileSigChecker.cs:142]: Good point

**Pull request merged**